lib/qb_integration/services/vendor.rb

```diff
@@ -29,9 +29,13 @@ def all(date, page, per_page)
       end
 
       def create
-        new_vendor = create_model
-        build new_vendor
-        @quickbooks.create new_vendor
+        begin
+          new_vendor = create_model
+          build new_vendor
+          @quickbooks.create new_vendor
+        rescue Quickbooks::IntuitRequestException => e
+          check_duplicate_name(e)
+        end
       end
```

Comment:
You can rescue the whole method, thusly:
```
def create
  <do whatever>
rescue
  <do whatever>
end
```

Reply:
I'll remember that for the future too!😀